create_infotexts.py

```diff
@@ -494,13 +489,13 @@ def process_region_addition_to_description(region_str, country_str):
 
         return region_addition
 
-    def add_catch_all_category(self, item):
+    def add_catch_all_category(self):
         """"
         Check if there are any content cats added and add generic content category to image.
         
         Populate self.content_cat with commons category, if present.
         """
-        if not self.content_cats:
+        if len(self.content_cats) == 0:
```

Comment:
That is because if doesn't just test for True/False but (in this case also interprets `None`,`[]`,`{}`,`0` and likely other empty elements as False.

The only one of these that might be very dangerous is `0`. I.e. if you are evaluating numbers and `num=0` is a valid number then you cannot use `if num` to guard agains `num is None`.

But that is not the case here so `if not self.content_cats:` should be used.